common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -299,6 +299,16 @@ trait FeatureSwitches {
     exposeClientSide = true,
   )
 
+  val AmpUSNewsSwitch = Switch(
+    SwitchGroup.Feature,
+    "amp-us-news-switch",
+    "If this switch is on, link to amp pages will be in the metadata for content tagged with us-news",
+    owners = Seq(Owner.withName("unknown")),
+    safeState = On,
+    sellByDate = never,
+    exposeClientSide = true,
+  )
```

Comment:
A feature switch is probably over the top for this test! Ignore this, I'll delete it